tala_connector.py

```diff
@@ -370,9 +373,9 @@ def _handle_list_projects(self, param):
         for item in data:
             action_result.add_data(data[item])
 
-        # Return success, no need to set the message, only the status
+        # Return success status with appropriate message
         # BaseConnector will create a textual message based off of the summary dictionary
-        return action_result.set_status(phantom.APP_SUCCESS)
+        return action_result.set_status(phantom.APP_SUCCESS, TALA_LIST_PROJECTS_SUCC)
```

Comment:
Please revert this. We are setting `Num Projects` in summary.

Reply:
Reverted.